src/common/snippets/include/snippets/pass_manager.hpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include "openvino/pass/manager.hpp"
+#include "openvino/pass/pass.hpp"
+#include "openvino/pass/validate.hpp"
+#include <typeinfo>
+
+
+namespace ov {
+namespace snippets {
+namespace pass {
+/**
+ * @brief Manager is like ov::pass::Manager, but allows to insert new passes at arbitrary places in the pipeline
+ * @ingroup snippets
+ */
+class Manager : public ov::pass::Manager {
+public:
+    ~Manager() override = default;
+    using PassBase = ov::pass::PassBase;
+    using Validate = ov::pass::Validate;
+    /**
+    * @brief PassPosition describes a particular position in a transformation pipeline,
+     * where a new transformation should be inserted.
```

Comment:
Minor: just for readability (and below)
```suggestion
    * @brief PassPosition describes a particular position in a transformation pipeline,
    *        where a new transformation should be inserted.
```

Reply:
Fixed, thanks